plugins/nid-team/rebase/rebase_manager.py

```
@@ -0,0 +1,702 @@
+                    return
+            elif decision == "squash":
+                log_info(f"Queueing configuration carry for squash: {sha} ({msg})")
+                squash_shas.append((sha, msg))
+            elif decision == "drop":
+                log_info(f"Skipping (dropping) carry: {sha} ({msg})")
+
+        # 4. Perform the squash of configuration carries
+        if squash_shas:
+            log_info(f"Cherry-picking and squashing {len(squash_shas)} configuration carries...")
+            # We cherry-pick them to get their contents onto the index
+            for sha, msg in squash_shas:
```

Ruff (0.16.1)

[warning] Loop control variable `msg` not used within loop body

(B007)